.github/actions/update_antora/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'Update Antora docs'
+description: 'Updates the Antora docs'
+inputs:
+  version:  
+    description: 'Full version'
+    required: true
+  major-minor-version:  
+    description: 'Minor version'
+    required: true
+  patch-version:  
+    description: 'Patch version'
+    required: true
+  is-pre-release:  
+    description: 'If the release is a pre-release version'
+    required: true
+  release-branch:  
+    description: 'Release branch'
+    required: true
+  custom-facet-old-version:  
```

Comment:
Q: Why are these inputs instead of just being derived from `version`?

Reply:
When updating the custom facet schemas, what we change differs between the pre release Antora docs update and the post release.